tests/series-upgrade/tests/bundles/xenial-to-bionic-queens-ha.yaml.j2

```diff
@@ -0,0 +1,648 @@
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  easyrsa:
+    charm: cs:~containers/easyrsa
+    num_units: 1
+
+  etcd:
+    series: bionic
+    #charm: cs:~containers/etcd-474
+    charm: cs:~containers/etcd
+    num_units: 3
+    options:
+      channel: 3.1/stable
+
+  glance:
+    charm: cs:~openstack-charmers-next/glance
+    num_units: 3
+    options:
+      openstack-origin: *openstack-origin
+      vip: {{ TEST_VIP04 }}
+    constraints: mem=1024
+
+  glance-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  gnocchi:
+    charm: cs:~openstack-charmers-next/gnocchi
+    num_units: 3
+    options:
+      openstack-origin: *openstack-origin
+      vip: {{ TEST_VIP05 }}
+
+  gnocchi-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  heat:
+    charm: cs:~openstack-charmers-next/heat
+    num_units: 3
+    options:
+      openstack-origin: *openstack-origin
+      vip: {{ TEST_VIP06 }}
+
+  heat-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  keystone:
+    charm: cs:~openstack-charmers-next/keystone
+    num_units: 3
+    options:
+      openstack-origin: *openstack-origin
+      vip: {{ TEST_VIP07 }}
+    constraints: mem=1024
+
+  keystone-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  memcached:
+    charm: cs:memcached
+    num_units: 1
+
+  percona-cluster:
+    charm: cs:~openstack-charmers-next/percona-cluster
+    constraints: mem=3072M
+    num_units: 1
+    options:
+      #vip: {{ TEST_VIP08 }}
+      source: *openstack-origin
+
+#  percona-hacluster:
+#    charm: cs:~openstack-charmers-next/hacluster
+#    num_units: 0
+#    options:
+#      cluster_count: 3
+
+  neutron-api:
+    charm: cs:~openstack-charmers-next/neutron-api
+    num_units: 3
+    options:
+      manage-neutron-plugin-legacy-mode: True
+      dns-domain: mojo.serverstack.
+      enable-ml2-dns: true
+      flat-network-providers: physnet1
+      ipv4-ptr-zone-prefix-size: 24
+      neutron-security-groups: true
+      openstack-origin: *openstack-origin
+      reverse-dns-lookup: true
+      vip: {{ TEST_VIP09 }}
+    constraints: mem=1024
+
+  neutron-gateway:
+    charm: cs:~openstack-charmers-next/neutron-gateway
+    num_units: 1
+    options:
+      bridge-mappings: physnet1:br-ex
+      instance-mtu: 1300
+      openstack-origin: *openstack-origin
+    constraints: mem=4096
+
+  neutron-hacluster:
+    charm: cs:~openstack-charmers-next/hacluster
+    options:
+      cluster_count: 3
+      corosync_transport: unicast
+
+  neutron-openvswitch:
+    charm: cs:~openstack-charmers-next/neutron-openvswitch
+
+#  placement:
```

Comment:
Also here: I don't understand why this is commented out. Inline explanation? If this is moved from somewhere else and you also don't know, that's fine, I won't bother you with this any longer ;) but if you know the reason then that would be really useful